app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -71,6 +51,11 @@ function AcuantCaptureCanvas() {
 
     cameraRef.current?.addEventListener('acuantcameracreated', onAcuantCameraCreated);
     return () => {
+      const canvas = document.getElementById('acuant-ui-canvas');
+      if (canvas) {
+        stopObservingProperty(canvas, 'callback');
+      }
+
       cameraRef.current?.removeEventListener('acuantcameracreated', onAcuantCameraCreated);
       resetObservableProperty(canvas, 'callback', originalDescriptor);
```

Comment:
Actually, we need to remove this line, and the definition of `resetObservableProperty` above (lines 10-28).
```suggestion
```

Reply:
Yup, addressed in 1e56c242730143ac719dda3e162b56da84ca43dd